worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/AbstractPlayerActor.java

```diff
@@ -334,7 +334,8 @@ public void floatAt(int x, int y, int z, boolean alwaysGlass) {
             BlockVector3 spot = BlockVector3.at(x, y - 1, z);
             final World world = getWorld();
             if (!world.getBlock(spot).getBlockType().getMaterial().isMovementBlocker()) {
-                try (EditSession session = WorldEdit.getInstance().getEditSessionFactory().getEditSession(world, 1, this)) {
+                try (EditSession session =
```

Comment:
is this break necessary? looks like you tried newline/indenting the original line and then converted to the shortcut method.